app/views/home/show.html+redesign.erb

```diff
@@ -0,0 +1,180 @@
+<%= open_graph_tags title: t(".title_og"), turbo_native_title: t(".turbo_native_title"), description: t(".description_og") %>
+
+<div class="bg-navy">
```

Comment:
I can't get the stylesheets to work with the variant setup correctly... More detail in the conversation thread.